integration/tests/tracer/test_tracer_debug_trace_complex_transactions.py

```diff
@@ -21,9 +22,9 @@
 
 
 @allure.feature("Tracer API")
-@allure.story("Tracer API RPC calls debug method trace_transaction iterative txs check")
+@allure.story("Tracer API RPC calls debug method trace_transaction iterative and scheduled txs check")
 @pytest.mark.usefixtures("accounts", "web3_client", "tracer_api", "tracer_validator")
-class TestDebugTraceIterativeTransaction:
+class TestDebugTraceIterativeAndScheduledTransactions:
```

Comment:
Scheduled Transactions can be iterative and non-iterative also
It is different categories